int32 id
float32 pixel_x
float32 pixel_y
float64 world_x
float64 world_y
float64 world_z
